### ISSUE TITLE
Discussion,general queries regarding Project

### ISSUE THREAD

**Issue by purnima143** [purnima143/Kurakoo#14]
Hello folks,
Thanks for showing your interest in this project😊. As you all know this project comes under **Girlscript Summer of Code 2021** open-source program, that's started today 🎉✨
Below are the following instructions, please read carefully:
- This project has nothing to do with existing directories and files. One issue will be created regarding the setup of the directories for both FRONT END and BACK END. Please,**don't create the same issue again and again**.
- **Designers**: Please create an issue with the proper issue title Eg: **"Design: Creating Feed page"**.Please work on this [figma](https://www.figma.com/file/1gYZlafa8bUZu61ji10unF/Kurakoo?node-id=0%3A1) file only otherwise your contribution won't accept.
- **Developers**: Please mention the field before creating an issue, if it's of front end write **"Front end: Login form"** in the issue title and similarly for the backend.
- **Contributors**: Everyone please create a valid issue name and branch. Before working on any issue kindly go through the instructions given in the [contributing](CONTRIBUTING.md) and [readme](README.md) file carefully.
- For any **queries**  comment below, use it as a group discussion inbox. If you guys wanna give any advice regarding the project then you are welcome🙂.
- Let's work on this project as a Team project😊👩‍💻👨‍💻✌

**Comment by Mayank0255** [comment]
@purnima143 
We can create proper issue templates for Designer and Developer type tickets

**Comment by pallavi-1812** [comment]
@purnima143 Hello! I would love to contribute to this project on the frontend part by using React JS but I am a little confused on how to contribute. Do I need to create a new issue or an issue will be already created and then I will need to work on that?

**Comment by saumyasrivastava08** [comment]
@purnima143 I want to contribute to this project but there are not certain issues can I create by my own and fix it.

**Comment by Mayank0255** [comment]
> @purnima143
> We can create proper issue templates for Designer and Developer type tickets

And while making those templates we can add respective labels on it like `gssoc21` as those are the ones which will be counted

**Comment by WebVishal** [comment]
Hey, I want to contribute to this project, but I'm not able to run it on my local system, anyone can provide some instructions.

**Comment by purnima143** [comment]
@pallavi-1812 Don't create an issue that is already there. Create a new issue if it doesn't exist

**Comment by purnima143** [comment]
@WebVishal In which field do you want to work on?

**Comment by purnima143** [comment]
> > @purnima143
> > We can create proper issue templates for Designer and Developer type tickets
> 
> And while making those templates we can add respective labels on it like `gssoc21` as those are the ones which will be counted

go ahead

**Comment by Mayank0255** [comment]
@purnima143 I don't have the right to do that. You would have to make me a maintainer of the project in such case

**Comment by purnima143** [comment]
@Mayank0255  I can't give access to you as a maintainer in this project. Or else work on other issues.

**Comment by SpecTEviL** [comment]
Hey @purnima143, Vishal Patil here and would like to inform you that I would like to work on making an automatic page slider component for this site under GSSoC'21. This slider would take the user to the top from the bottom with the help of just a single button. If you think this issue is appropriate you can make this issue and assign it to me or else if needed I can make it and then you can assign it to me. For reference, you can see the one I have on my portfolio https://spectevil.github.io in the right bottom corner. 
Awaiting your reply. 
Thanks and regards, 
Vishal Patil.

**Comment by purnima143** [comment]
> Hey @purnima143, Vishal Patil here and would like to inform you that I would like to work on making an automatic page slider component for this site under GSSoC'21. This slider would take the user to the top from the bottom with the help of just a single button. If you think this issue is appropriate you can make this issue and assign it to me or else if needed I can make it and then you can assign it to me. For reference, you can see the one I have on my portfolio https://spectevil.github.io in the right bottom corner.
> Awaiting your reply.
> Thanks and regards,
> Vishal Patil.

@SpecTEviL  Create an issue for that

**Comment by SpecTEviL** [comment]
> > Hey @purnima143, Vishal Patil here and would like to inform you that I would like to work on making an automatic page slider component for this site under GSSoC'21. This slider would take the user to the top from the bottom with the help of just a single button. If you think this issue is appropriate you can make this issue and assign it to me or else if needed I can make it and then you can assign it to me. For reference, you can see the one I have on my portfolio https://spectevil.github.io in the right bottom corner.
> > Awaiting your reply.
> > Thanks and regards,
> > Vishal Patil.
> 
> @SpecTEviL Create an issue for that

Sure @purnima143 will do that ASAP.

**Comment by vdmondkr2002** [comment]
> For backend discussion pls come to the **#kurakoo** in **GSSOC 21** server on **Discord**. Otherwise, it would create conflicts among us and we couldn't work on the project properly.

@purnima143 I also want to contribute this project's backend but i am not able to find the channel **#kurakoo** in **GSSOC 21** server, Have they given access?

**Comment by purnima143** [comment]
@vdmondkr2002  Not  yet

**Comment by WebVishal** [comment]
> @WebVishal In which field do you want to work on?

hi @purnima143  I want to Contribute in the frontend, actually when I putting npm start cmd then it shows an error missing script: start

**Comment by purnima143** [comment]
> > @WebVishal In which field do you want to work on?
> 
> hi @purnima143 I want to Contribute in the frontend, actually when I putting npm start cmd then it shows an error missing script: start

Did you instal npm?

**Comment by WebVishal** [comment]
> Did you instal npm?

yes i have installed npm after that i again run npm start the it show 
![IMG_20210313_190819](https://user-images.githubusercontent.com/76445546/111032145-d7dfd800-8430-11eb-9c23-02b71cd61a29.jpg)

**Comment by purnima143** [comment]
> > Did you instal npm?
> 
> yes I have installed npm after that i again run npm start then it shows
> ![IMG_20210313_190819](https://user-images.githubusercontent.com/76445546/111032145-d7dfd800-8430-11eb-9c23-02b71cd61a29.jpg)

1. Never use C drive change the drive of the project
2. You have to go to the client folder
3. Then  npm install
4. npm start

**Comment by pallavi-1812** [comment]
Hello @purnima143 , I want to contribute to frontend but after cloning the repository and running it locally, I am getting a blank page and an error in console saying 
Store does not have a valid reducer. Make sure the argument passed to combineReducers is an object whose values are reducers.
console.<computed> @ index.js:1

**Comment by purnima143** [comment]
> Store does not have a valid reducer. Make sure the argument passed to combineReducers is an object whose values are reducers.

@pallavi-1812 The blank page is fine bcoz we don't have a home page now only signup/in and feedpage. I didn't get such type of errors. Let me check
